src/clients/python/__init__.py

```diff
@@ -987,6 +987,12 @@ def _prepare_request(self, inputs, outputs,
                              "' values must be specified as a list of numpy arrays" \
                              " or as a single c_void_p representing the shared memory handle")
             if type(inp) != c_void_p:
+                flag = False
```

Comment:
What is this checking for? Put a comment. Also just have a compound if statement that has "continue" as the body.

Reply:
Checks for the case in variable size tensors when (handle, tensor size) is passed